docs/concepts/pydantic_prompts.md

```diff
@@ -104,10 +106,15 @@ Can you recommend some books on the following topic and genre pairs?
 
 By default, the `Prompt` class treats the prompt template as a single user message. If you want to specify a list of messages instead, we provide a decorator to make this easy:
 
+!!! note 
+	
+	`@messages` decorator adds `messages` property to the class
+
 ```python
+from mirascope import Prompt
```

Comment:
I've been thinking about this and am not sure of the answer.

When importing messages, we can technically do:

`from mirascope.prompts import messages, Prompt`

instead of having separate imports. What is your opinion here?

Reply:
i like one liner, will change